src/pretix/presale/templates/pretixpresale/event/order.html

```diff
@@ -445,45 +447,31 @@ <h3 class="panel-title">
                                     {% trans "This will invalidate all tickets in this order." %}
                                 </p>
                             {% endif %}
-                            <p>
-                            <a href="{% eventurl event 'presale:event.order.cancel' secret=order.secret order=order.code %}"
-                                    class="btn btn-danger">
-                                <span class="fa fa-remove" aria-hidden="true"></span>
-                                {% trans "Cancel order" %}
-                            </a>
-                            </p>
                         {% else %}
                             <p>
-                                {% if order.total != 0 and order.user_cancel_fee %}
+                                {% if request.event.settings.cancel_terms_unpaid %}
+                                    {{ request.event.settings.cancel_terms_unpaid|rich_text }}
+                                {% elif order.total != 0 and order.user_cancel_fee %}
                                     {% blocktrans trimmed with fee=order.user_cancel_fee|money:request.event.currency %}
                                         You can cancel this order. As per our cancellation policy, you will still be required
                                         to pay a cancellation fee of <strong>{{ fee }}</strong>.
                                     {% endblocktrans %}
+                                    {% trans "This will invalidate all tickets in this order." %}
                                 {% else %}
                                     {% blocktrans trimmed %}
                                         You can cancel this order using the following button.
                                     {% endblocktrans %}
+                                    {% trans "This will invalidate all tickets in this order." %}
                                 {% endif %}
-                                {% trans "This will invalidate all tickets in this order." %}
-                            </p>
-
-                            {% if order.user_cancel_deadline %}
```

Comment:
Has this been removed intentionally? It seems useful?

Reply:
I moved it to the automatically generated cancelation terms to lines 459 and 464 as I thought one might want to overwrite _everything_ when adding custom terms. But you are right, maybe this is nonetheless useful to note and might easily be forgotten in custom terms. I can move back in if you want.

Reply:
Sorry, I was refering to the `{% if order.user_cancel_deadline %}` block. I agree that this should be overwritten by custom terms! But I only see it removed from the div, not added somewhere else?

Reply:
Ah, I did not see the line you are referring to as changed. Are you sure you are on the latest commit on that branch? Super strange.

Reply:
Huh. It was showing it to me multiple times. GitHub :shrug: Now it's clean